library/psa_crypto_slot_management.c

```diff
@@ -232,4 +232,36 @@ psa_status_t psa_close_key( psa_key_handle_t handle )
     return( psa_wipe_key_slot( slot ) );
 }
 
+void mbedtls_psa_get_stats( mbedtls_psa_stats_t *stats )
+{
+    psa_key_handle_t key;
+    memset( stats, 0, sizeof( *stats ) );
+    for( key = 1; key <= PSA_KEY_SLOT_COUNT; key++ )
+    {
+        psa_key_slot_t *slot = &global_data.key_slots[key - 1];
```

Comment:
cache_slots are not set here. Should they?

Reply:
I put `cache_slots` in for future use. We don't have them currently, so they stay at 0.